Apps/Inc/Tasks.h

```diff
@@ -177,15 +158,53 @@ extern task_trace_t PrevTasks;
 /**
  * Error variables
  */
-extern fault_bitmap_t FaultBitmap;
 extern os_error_loc_t OSErrLocBitmap;
 
+// Store error codes that are set in task error assertion functions
+extern error_code_t Error_ReadTritium; 
+extern error_code_t Error_ReadCarCAN;
+extern error_code_t Error_UpdateDisplay;
+
+/**
+ * Error-handling option enums
+*/
+
+// Scheduler lock parameter option for asserting a task error
+typedef enum {
+    OPT_LOCK_SCHED = false,
+    OPT_NO_LOCK_SCHED = true
```

Comment:
If this is for passing the value in as an argument to an OS function, it makes sense, just maybe put a comment explaining here.

Reply:
Ah, this is for passing an argument into the task error function that I wrote, so I can change it to use default enum values where `OPT_NO_LOCK_SCHED` = 0 instead. I think I did it this way because I was imagining a yes/no question where the default locks the scheduler, but it was mostly an arbitrary choice that now seems more confusing than it needs to be.

Reply:
Gotcha